src/editpage/components/blocks/WpBlock.vue

```diff
@@ -44,7 +44,13 @@ export default {
       }
     },
     render: function(text) {
-      return md.render(text);
+      // Replace coqdoc-style headers (*) with markdown headers (#)
+      let converted = text.replace(/^[ ]*([*]+) /gm, (match, p1) => {
```

Comment:
Should we convert this everytime?
Or maybe just once at importing.
Or is this to preserve the original content of the file for saving.

Reply:
Yes, one reason is for preserving original content. If we could guarantee that we can convert back, then that would also be okay, but I think that the conversion is currently not good enough to guarantee that. Another reason is that this way we could write in coqdoc style markdown, which stays closer to the coqdoc ecosystem.

We may want to formalize guarantees about exporting and importing (that importing and then exporting does not change the file for instance), but I thought of postponing that to a later stage.

Reply:
In that case this is fine, I also forgot that the actual block 'cache' this value because it is used as computed values.
So I think we can merge this.